hydromt_wflow/wflow.py

```diff
@@ -4357,10 +4370,10 @@ def setup_other_demand(
         variables: list = ["ind_gross", "ind_net", "lsk_gross", "lsk_net"],
```

Comment:
updated hydromt names
```suggestion
        variables: list = ["industry_gross", "industry_net", "livestock_gross", "livestock_net"],
```